haas/api.py

```diff
@@ -22,6 +22,7 @@
 from schema import Schema, Optional
 
 from haas import model
+from haas.auth import get_auth_backend
```

Comment:
Ok. I added this to newly created issue #540 so that we can track it. I also updated the coding guidelines in #511 to prefer removing redundant stuff.